compiler/crates/relay-lsp/src/find_field_usages/find_field_locations.rs

```diff
@@ -19,6 +19,7 @@ pub(crate) fn find_field_locations(
     type_name: StringKey,
 ) -> Option<Vec<IRLocation>> {
     let type_ = program.schema.get_type(type_name)?;
+    // todo: this doesn't find usages of renamed RelayResolver fields
```

Comment:
The renaming of a Relay Resolver field only works once. After the rename there aren't any more references returned from this FieldUsageFinder.

I believe this is because the rename of the field happens without user interaction, so there are some caches and so that aren't updated as they usually would. We could probably trigger the ingestion of changes from within the rename code, but I don't know how at the moment.
Maybe one of you can quickly help out?